src/lkl/posix-host.c

```diff
@@ -18,9 +18,9 @@
 #include "lkl/posix-host.h"
```

Comment:
What are all of the whitespace changes in this file for?  They appear to be making the style inconsistent with the rest of the file (which uses the Linux coding style, because it is based on a file from LKL).

Reply:
Likely my auto-formatter. I think everything will end up in OE format anyways, so no point in making exceptions now.

Reply:
This file shouldn't be, so that we can diff it against the upstream version.

Reply:
Makes sense, I'll clean it up then. Does that apply to all files in `src/lkl`?

Reply:
It shouldn't.  This is the only one copied from the LKL tree, as far as I'm aware.

Reply:
Let's add a `// clang-format off` annotation at the top of the file to avoid this in the future.

Reply:
Done.